pydantic/main.py

```diff
@@ -123,6 +126,7 @@ class BaseConfig:
     json_loads: Callable[[str], Any] = json.loads
     json_dumps: Callable[..., str] = json.dumps
     json_encoders: Dict[Type[Any], AnyCallable] = {}
+    underscore_attrs_are_private = False
```

Comment:
```suggestion
    underscore_attrs_are_private: bool = False
```
I know it's obvious, but best to be explicit.